npm/ng-packs/packages/account/src/lib/account-routing.module.ts

```diff
@@ -14,9 +14,9 @@ import { ManageProfileComponent } from './components/manage-profile/manage-profi
 import { RegisterComponent } from './components/register/register.component';
 import { ResetPasswordComponent } from './components/reset-password/reset-password.component';
 import { eAccountComponents } from './enums/components';
-import { AccountExtensionsGuard, AuthenticationFlowGuard } from './guards';
+import { AccountExtensionsGuardFn, AuthenticationFlowGuardFn } from './guards';
 
-const canActivate = mapToCanActivate([AuthenticationFlowGuard]);
+const canActivate = [AuthenticationFlowGuardFn];
```

Comment:
why there is here.  ? We didn't use anywhere

Reply:
https://github.com/abpframework/abp/blob/069dcd9c86c1d344d190f793e6b6c36a53286949/npm/ng-packs/packages/account/src/lib/account-routing.module.ts#L30C21-L30C21

it's used several places